testsuite/features/step_definitions/setup_wizard_steps.rb

```diff
@@ -0,0 +1,12 @@
+When(/^I enter the scc credentials$/) do
+  user = ENV['SCC_CREDS'].split('|')[0]
+  password = ENV['SCC_CREDS'].split('|')[1]
```

Comment:
`ENV['SCC_CREDENTIALS']`

Reply:
Wouldn't it be preferable to have two separate variables?

Reply:
That was my plan but eric said one.

Reply:
Reason for having only one:
* we might run the test suite manualy, and it's easier to set up one environment variable than two
* testing a tag at each scenario slows down the test suite, so it's better to have only one tag
* a scenario marked with only one tag is more readable

The only drawback is that it requires to unmarshal the variable when filling in the form: this happens only once in the whole test suite.

Please also note that Michael took same approach for his 3 portus variables.

Reply:
Are tags necessarily bound to environment variables? I thought they were independent concepts.

Reply:
They are, but for simplicity sake, we are trying to keep strict parallelism.